docs_user/modules/proc_migrating-ovn-data.adoc

```diff
@@ -52,7 +68,12 @@ apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
   name: ovn-data-cert
+ifeval::["{build_variant}" != "ospdo"]
   namespace: openstack
```

Comment:
Some uses of `openstack` namespace are parametrized while others are hardcoded. We should be consistent.